### PR TITLE
Rspec fake payment

### PR DESCRIPTION
**Pull request by Nazarii-Makohin**

### CODE REVIEW

File: spec/factories/event.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
     title { Faker::Movie.unique.title}
     description { Faker::Lorem.paragraph(sentence_count: 3) }
     duration { Faker::Number.number(digits: 3) }
-    event_entry_tickets { Faker::Number.number(digits: 3) }
+    event_entry_tickets { 2 }
```
Comment (Nazarii-Makohin) [inline spec/factories/event.rb:6]: what is that for?

Comment (Nazarii-Makohin) [reply spec/factories/event.rb:6]: dont push it to origin, use local\

File: spec/factories/orders.rb [hunk 2]
```diff
@@ -1,5 +1,10 @@
 FactoryBot.define do
   factory :order do
-    
+    title { Faker::Movie.unique.title }
+    user_id { 1 }
```
Comment (olegsobchuk) [inline spec/factories/orders.rb:4]: suppose `user_id` define relation to `User` model. Are you sure that this works properly?

File: spec/factories/orders.rb [hunk 3]
```diff
@@ -1,5 +1,10 @@
...
+    event_id { 1 }
+    order_amount { Faker::Number.number(digits: 3) }
```
Comment (olegsobchuk) [inline spec/factories/orders.rb:6]: why do you use `Faker` for tests? Use pure ruby way for this it will be faster

**Pull request merged**